### DISCUSSION TITLE
Is there a way to see which graphql route get executed in the logs

### DISCUSSION THREAD

**Discussion by sanojsilva** [strapi/strapi#7868] [Legacy Topics]
Currently we get something like this
![image](https://user-images.githubusercontent.com/31297777/93002597-c7b87000-f555-11ea-98f5-11d25c7d9577.png)

**Comment by derrickmehaffy** [comment]
There is not currently no, I would suggest you make a feature request to add in some additional logging for the GraphQL plugin or feel free to submit a PR yourself. Thank you!

**Reply by sanojsilva** [reply]
Yes I'll look into that

**Reply by lauriejim** [reply]
Hello! Can you give me more details about your exact need, I'm not sure to understand what you want to display.
Thank you.
